hera_cal/frf.py

```diff
@@ -10,16 +10,26 @@
     HAVE_UVTOOLS = False
 
 from . import utils
-
+import scipy.interpolate as interp
 from .datacontainer import DataContainer
 from .vis_clean import VisClean
-from pyuvdata import UVData, UVFlag
+from pyuvdata import UVData, UVFlag, UVBeam
 import argparse
 from . import io
 from . import vis_clean
 import warnings
+from astropy.coordinates import SkyCoord, EarthLocation, AltAz, ITRS
+import astropy.units as units
 import astropy.constants as const
-
+import healpy as hp
+from astropy.time import Time
+from pyuvdata import utils as uvutils
+from . import utils
+import copy
+from .utils import echo
+import datetime
+import astropy.constants as const
+from . import redcal
 
 SPEED_OF_LIGHT = const.c.si.value
 SDAY_KSEC = 86163.93 / 1000.
```

Comment:
Since you've imported `astropy.units`, wouldn't it make more sense to replace this with `SDAY_KSEC = units.sday.to("ks")`? On that note, it looks like the value here doesn't exactly agree with what `astropy` says (though it's off by less than a second).